client/packages/core/src/instaql.js

```diff
@@ -690,6 +690,12 @@ function resolveObjects(store, { etype, level, form, join, pageInfo }) {
 
   const limit = form.$?.limit || form.$?.first || form.$?.last;
   if (limit != null) {
+    if (level > 0) {
+      console.warn(
+        'WARNING: Limits in child queries are only run client-side. Data returned from the server will not have a limit.',
```

Comment:
I go back and forth on the warning. I guess let's just start with a warning, as it is the most explicit.